corehq/apps/cachehq/cachemodels.py

```diff
@@ -97,14 +97,6 @@ class DomainInvitationGenerationCache(GenerationCache):
     ]
 
 
-class CommtrackConfigGenerationCache(GenerationCache):
```

Comment:
also I think based on the way the caching stuff works you should be able to just change the view name and leave this (since I believe that's only used in invalidation). not 100% sure about that though.